cli/mender-artifact/artifacts.go

```diff
@@ -32,6 +32,27 @@ import (
 	"github.com/urfave/cli"
 )
 
+type writerUpdateStorer struct {
```

Comment:
It seems it is customary in go to name this `writeUpdateStorer` (I'm feeling pedantic today.)

Reply:
Alright, fixed.